src/test/java/io/confluent/copycat/jdbc/JdbcSourceTaskTestBase.java

```diff
@@ -26,18 +28,26 @@
 
 import io.confluent.common.utils.MockTime;
 import io.confluent.common.utils.Time;
-import io.confluent.copycat.data.Schema;
-import io.confluent.copycat.data.SchemaBuilder;
-import io.confluent.copycat.source.SourceTaskContext;
-import io.confluent.copycat.storage.OffsetStorageReader;
 
 public class JdbcSourceTaskTestBase {
 
   protected static String SINGLE_TABLE_NAME = "test";
+  protected static Map<String, Object> SINGLE_TABLE_PARTITION = new HashMap<>();
+
+  static {
+    SINGLE_TABLE_PARTITION.put("table", SINGLE_TABLE_NAME);
```

Comment:
The constant "table" is used all over the place. Refactor it into a constant in one of the Jdbc* classes?